### PR TITLE
Post a new comment to an existing article functionality 

### PR DESCRIPTION
**Pull request by AnnaTas77**

### CODE REVIEW

File: src/components/CommentAdder.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <div className="form-container">
+            <form className="comment-adder-form" onSubmit={handleFormSubmit}>
+                <label htmlFor="comment-text">Add a new comment</label>
+                <textarea
+                    id="comment-text"
+                    type="text"
+                    required
+                    maxLength="250"
+                    value={newComment.body}
+                    onChange={(event) => {
```
Comment (NRMSMITH) [inline src/components/CommentAdder.jsx]: I would suggest extrating this function out for readability :)

File: src/components/CommentAdder.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                console.log(err);
+            });
+
+        setNewComment({
+            username: "grumpy19",
+            body: "",
+        });
+    };
+
+    console.log("newComment", newComment);
```
Comment (NRMSMITH) [inline src/components/CommentAdder.jsx]: please remove logs! :)

File: src/components/CommentAdder.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    const handleFormSubmit = (event) => {
+        event.preventDefault();
+        postNewComment(articleId, newComment)
+            .then((postedCommentFromDB) => {
+                setComments((currentComments) => {
+                    return [postedCommentFromDB, ...currentComments];
+                });
+            })
+            .catch((err) => {
+                console.log(err);
```
Comment (NRMSMITH) [inline src/components/CommentAdder.jsx:38]: so this console log doesn't really serve the user - maybe perhaps you could set a boolean state to indicate if there is an error, and then conditionally render a message to the user when there is an error?

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
Have a rethink about how the user sees they've made a mistake :D

**Review by xViolaine — APPROVED** [review]
Great job taking the given feedback on board!

**Pull request merged**